apps/fabric-website/index.html

```diff
@@ -57,7 +60,9 @@
 
     // If the site is hosted, load the flight config script for production or dogfood
     if (!isLocal) {
-      var configPrefix = isProduction ? 'fabric-website-prod' : 'fabric-website-df';
+      var v5 = fabricVersion === '5' ? 'v5-' : '';
```

Comment:
Agreed with @KevinTCoughlin's callout. @leddie24, this check could be made more durable--for example, have something like a map/array of supported versions and check if the query param is in it. If yes, use it; if not, fall back to latest.

Reply:
Added an array with supported versions.   If the support version exists in the array and isn't the latest version, then it adds the prefix modifier to the `configPrefix`.